### PR TITLE
Ports 'Pointing at something on yourself now shows the item'

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Ports https://github.com/tgstation/tgstation/pull/68642

## Why It's Good For The Game
It's nice to be able to point at something without having to throw it on the floor.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

https://user-images.githubusercontent.com/110184118/220732534-dbc2fb7a-6b10-453b-b11d-243d56a459a0.mp4

</details>

## Changelog
:cl:Mothblocks, CydiaLamiales
add: Pointing at something on yourself now shows the item.
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
I swore this was in an earlier pr?

Did that never get merged? 
Hm

**Comment by CydiaLamiales** [comment]
> I swore this was in an earlier pr?
> 
> Did that never get merged? Hm

Dunno, honestly.

**Comment by CydiaLamiales** [comment]
Yes is this now possible
image

**Comment by DrCrawler** [comment]
You should make it so if their item is in a pocket/its a circlehand that it just notifies people around you and can only be seen if they are standing near you and examine. Would be awesome for pranks.

**Comment by itsmeow** [comment]
the bubble is weird, it should just be like a transparent render of the item

**Review by itsmeow — COMMENTED** [review]
Code is fine, the bubble looks weird though

**Comment by CydiaLamiales** [comment]
Nope nope nope ABORT

File: html/changelog.html [hunk 1]
```diff
@@ -87,6 +87,7 @@ <h3 class="author">itsmeow updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">Fixed map and shuttle votes not properly reacting to votes.</li>
 				<li class="server">Update to Python 3.11.2, Node 18.14.2, Inferno 8, Typescript 4.9, Yarn 3.4</li>
+				<li class="rscadd">Added fallback species config option, for use when there is no database attached.</li>
```
Comment (CydiaLamiales) [inline html/changelog.html]:
```suggestion
```

**Comment by CydiaLamiales** [comment]
New NEW icon
image

**Review by itsmeow — APPROVED** [review]
Looks better, yeah. Code looks fine, pretty simple.

**Pull request merged**